.github/workflows/ci.yml

```diff
@@ -78,14 +80,17 @@ jobs:
           distribution: 'temurin'
           cache: 'sbt'
       - uses: sbt/setup-sbt@v1
-      - run: sbt '++${{ matrix.scala }}!; testsJVM/test'
+      - name: Test Parser on ${{ matrix.scala }}
+        run: sbt '++${{ matrix.scala }}!; testsJVM/test'
+      - name: Test Semanticdb on ${{ matrix.scala }}
+        run: sbt ++${{ matrix.scala }} testsSemanticdb/test
   windows:
     name: Windows tests
     runs-on: windows-latest
     steps:
       - uses: actions/checkout@v4
       - uses: olafurpg/setup-scala@v14
-      - run: sbtx ci-windows
+      - run: sbtx "testsJVM/test; testsSemanticdb/test"
```

Comment:
stupid question: why in one case you create separate `run` entries for `JVM` vs `Semanticdb` tests, and in another you invoke them together?

Reply:
I added some nice names above and forgot to do it, I will add it here as well.